app/src/org/commcare/utils/UriToFilePath.java

```diff
@@ -72,6 +77,10 @@ public static String getPathFromUri(final Context context, final Uri uri) {
                 return getDataColumn(context, contentUri, selection, selectionArgs);
             }
         } else if ("content".equalsIgnoreCase(uri.getScheme())) {
+            // Return the remote address
+            if (isGooglePhotosUri(uri)) {
```

Comment:
Is this from SO? Or can you link to docs about why this is handled specially?

Reply:
This is result of merging FileUtil::getPath in here. Corresponding code https://github.com/dimagi/commcare-android/blob/master/app/src/org/commcare/utils/FileUtil.java#L622